src/components/layout/yearInReview/2021/partnerships.tsx

```diff
@@ -35,90 +37,119 @@ const Partnership: React.FC<PartnershipProps> = ({
-              practice footage to edify the public about animal exploitation.
-              They hold street activism events worldwide to edify the public
-              towards supporting animal rights, living vegan and speaking up for
-              the animals.
+              <FormattedMessage
+                id="page.year-in-review.2021.section.partnerships.anonymous-for-the-voiceless.paragraph.0"
+                defaultMessage="<no-localization>Anonymous for the Voiceless (AV)</no-localization> is a not-for-profit animal rights organisation specializing in using conversation and standard practice footage to edify the public about animal exploitation. They hold street activism events worldwide to edify the public towards supporting animal rights, living vegan and speaking up for the animals."
+              />
             </p>
             <p>
-              Traditionally, it&apos;s been very hard to track the effectiveness
-              of street outreach. We&apos;re excited to have partnered up with
-              AV in order to be able to provide them the tools they need to
-              monitor and track the effectiveness of each group with{' '}
-              <CustomLink href="https://activisthub.org">
-                Activist Hub
-              </CustomLink>
-              .
+              <FormattedMessage
+                id="page.year-in-review.2021.section.partnerships.anonymous-for-the-voiceless.paragraph.1"
+                defaultMessage="Traditionally, it's been very hard to track the effectiveness of street outreach. We're excited to have partnered up with <no-localization>AV</no-localization> in order to be able to provide them the tools they need to monitor and track the effectiveness of each group with <link>Activist Hub</link>."
+                values={{
+                  link: (chunks) => (
+                    <CustomLink href="https://activisthub.org">
+                      {chunks}
+                    </CustomLink>
+                  ),
+                }}
+              />
             </p>
           </Partnership>
-          <Partnership image={otherPartner} title="Other Partnerships">
+          <Partnership
+            image={otherPartner}
+            title={intl.formatMessage({
+              id: 'page.year-in-review.2021.section.partnerships.other-partnerships.heading',
+              defaultMessage: 'Other Partnerships',
+            })}
+          >
             <p>
               <CustomLink href="https://veganoutreach.org">
                 <span className="block md:inline">
                   <b>Vegan Outreach</b>
                 </span>
               </CustomLink>
               <span className="hidden md:inline">&nbsp;&mdash;&nbsp;</span>
-              We partnered with Vegan Outreach&apos;s Vegan Mentorship Program
-              in order to provide completely free mentorship access to all Vegan
-              Bootcamp and 3Movies.org participants.
+              <FormattedMessage
```

Comment:
Please add a FormattedMessage for the above Vegan Outreach link.

Reply:
Isn't Vegan Outreach an organisation so we wouldn't want it translated? Or would you want a `defaultMessage` that's just `defaultMessage="<no-localization>Vegan Outreach</no-localization>"` to better facilate future changes or something?

Reply:
The latter, text changes should be done easily by anyone without having to dig through the code.